katalog/vsphere-cm/vsphere-cloud-controller-manager.yaml

```diff
@@ -216,14 +216,17 @@ spec:
         tier: control-plane
     spec:
       nodeSelector:
-        node-role.kubernetes.io/master: ""
+        node-role.kubernetes.io/control-plane: ""
```

Comment:
Do all the Kubernetes versions supported by this module, support this "new" label?

Reply:
Yes, 1.23 and 1.24 have "control-plane" as node-role